linkinator.config.json

```diff
@@ -1 +1 @@
-{"recurse":true,"skip":["https://codecov.io/gh/googleapis/","www.googleapis.com","img.shields.io"],"silent":true,"concurrency":10}
\ No newline at end of file
+{"recurse":true,"skip":["https://github.com/googleapis/nodejs-appengine-admin/blob/master/CHANGELOG.md", "https://codecov.io/gh/googleapis/","www.googleapis.com","img.shields.io"],"silent":true,"concurrency":10}
```

Comment:
Could we dig into what's creating this file?  It should be well formatted JSON.

Reply:
Oh, that's the library automation tool I built. I need to add formatting. Sorry!